examples/zlib_tokio_02_write.rs

```diff
@@ -0,0 +1,31 @@
+use async_compression::tokio_02::write::ZlibDecoder;
+use async_compression::tokio_02::write::ZlibEncoder;
+
+use std::io::Result;
+use tokio_02::io::AsyncWriteExt; // for `write_all` and `shutdown`
+use tokio_02::runtime::Runtime;
+
+async fn async_main() {
```

Comment:
Can you add the `macros` feature to the `tokio` dependency, and use `?` instead of `unwrap` everywhere

```suggestion
#[tokio::main]
async fn main() -> std::io::Result<()> {
```
async fn async_main() {
```

Reply:
will fix

Reply:
fixed it